web-view/bootstrap/src/ViewerBootstrapper.ts

```diff
@@ -1,28 +1,29 @@
-const {AppOptions} = require("pdfjs-dist/lib/web/app_options");
-const {PDFViewerApplication} = require("pdfjs-dist/lib/web/app");
-import {GenericExternalServices} from "./support/GenericExternalServices"
+// @ts-ignore
+import { AppOptions } from "pdf.js/web/app_options";
+Object.defineProperty(window, "PDFViewerApplicationOptions", {
+  get: () => AppOptions
+});
+const {PDFViewerApplication} = require("pdf.js/web/app");
 import {getViewerConfiguration} from "./support/ViewerConfiguration";
-import "pdfjs-dist/lib/web/pdf_print_service";
+import "pdf.js/web/pdf_print_service";
 
 export class ViewerBootstrapper {
   static defineViewer(): any {
-    PDFViewerApplication.externalServices = GenericExternalServices;
     Object.defineProperty(window, "PDFViewerApplication", {
       get: () => PDFViewerApplication
     });
-    AppOptions.set("workerSrc", "pdf.worker.js");
+    AppOptions.set("workerSrc", "pdf.worker.mjs");
     AppOptions.set("cMapUrl", "cmaps/");
-    Object.defineProperty(window, "PDFViewerApplicationOptions", {
-      get: () => AppOptions
-    });
     return PDFViewerApplication;
   }
 
   static load(fileUrl: String | null = null): Promise<void> {
     return new Promise(resolve => {
       AppOptions.set("defaultUrl", fileUrl);
       const config = getViewerConfiguration();
-      config.eventBus.on("documentloaded", () => resolve());
+      PDFViewerApplication.initializedPromise.then(function () {
```

Comment:
see https://github.com/mozilla/pdf.js/wiki/Third-party-viewer-usage#initialization-promise

Reply:
and https://github.com/mozilla/pdf.js/commit/08256e6795bbec885746cba1c198c0b4c5f84982